### ISSUE TITLE
`  title String @db.VarChar(255)` leads to `Attribute not know: "@db.VarChar"`

### ISSUE THREAD

**Issue by janpio** [Ovyerus/prismaliser#2]
Currently for default schema on https://prismaliser.ovy.cloud/

![image](https://user-images.githubusercontent.com/183673/127317921-36f005fd-1cfe-4964-896a-fd6166f7fc99.png)

Removing the native type definition fixes the problem of course.

Maybe `db` is not defined as the `datasource` name that is sent in as validation? Could be that you have to add that (or just include it in the default schema for the validation to be able to pick it up from there).

Update: Yep, adding this to the default schema makes it work:

```
datasource db {
  provider = "mysql"
  url = "mysql://foo"
}
```

**Comment by janpio** [comment]
Update 2: Maybe this is just using a cached schema vs. the current default that already would fix this problem?

**Comment by Ovyerus** [comment]
Yeah it's the cached schema. I fixed this earlier during testing the fix for #1 while I came across it, but it means that previous users still have the old one cached
